code_samples/api/product_catalog/src/Command/CatalogCommand.php

```diff
@@ -0,0 +1,90 @@
+<?php declare(strict_types=1);
```

Comment:
```suggestion
<?php 

declare(strict_types=1);
```

Please apply this suggestion to all commands.